orchard/orchard_runner.go

```diff
@@ -37,3 +38,17 @@ func parseCommandLine(command string) ([]string, error) {
 	err := json.Unmarshal([]byte(command), &output)
 	return output, err
 }
+
+// parse artifact expecting a s3 URI to return the bucket and path
+func (c OrchardStdoutRunner) ParseArtifact(artifact string) (string, string, error) {
```

Comment:
is this called some where? I cannot seem to find it

Reply:
not called yet.   I plan to use it in the next PR.

Reply:
I have taken it out.

Reply:
or maybe add the next part to show a very simplified usage?

Reply:
I have take out `ParseArtifact` in this PR.  Better to separate it in the next PR with the rest of the logic.  I plan to have a different struct `OrchardJavaRunner` then.

Reply:
ok,  I'll merge this one this time. But it's better to try implement complete (but minimum viable) feature in PRs. Those features can be very lean with rough edges, but having unused code like `simple_s3.go` in a service repo is not desired state. Not much of the changes in this PR have any real usage other than providing utilities.